tools/gmhazard_scripts/db_creation/empirical_db/empirical_model_configs/21p10.yaml

```diff
@@ -36,14 +36,14 @@ ACTIVE_SHALLOW:
       - CY_14
   PGA:
     geom:
-      - Br_10  # ZA_06 or Br_10 or BSSA_14 or CB_14 or CY_14 or ASK_14
+      - Br_10 # ZA_06 or Br_10 or BSSA_14 or CB_14 or CY_14 or ASK_14
```

Comment:
![image](https://user-images.githubusercontent.com/25143301/138357135-f946590b-9d98-4772-9d26-966777dcbcd9.png)
Unsure if pep8 is required for yaml but pycharm at least complains about it to me.

Reply:
Black with PyCharm didn't work so I used VSCode's prettier :joy: 
Adding space now and remove that comment.